packages/dd-trace/src/appsec/waf/waf_context_wrapper.js

```diff
@@ -27,31 +28,37 @@ class WAFContextWrapper {
 
     const payload = {}
     let payloadHasData = false
-    const inputs = {}
+    const persistentInputs = {}
+    const ephemeralInputs = {}
```

Comment:
yes but now that you moved the rest of the if blocks, those are way overscoped, you can move them in their respective `if( presistent !== null ...`